ingest-app/src/cmr/ingest/api/subscriptions.clj

```diff
@@ -18,6 +19,13 @@
     (v/validate-concept-metadata concept)
     concept))
 
+(defn subscription-user-matches-token?
+  "Check to see if the user for the supplied token matches the user in the subscription request"
+  [concept request-context headers]
+  (let [token-user (api-core/get-user-id request-context headers)
+        subscriber (get (json/decode (:metadata concept)) "SubscriberId")]
```

Comment:
We normally use symbols to parse out JSON. change this to:
```suggestion
        subscriber (:SubscriberId (json/decode (:metadata concept) true))]
```

Reply:
I didn't think the key was a symbol, I thought it was a string.